### ISSUE TITLE
Test/Run/Publish to web wont build project

### ISSUE THREAD

**Issue by masterboltdev** [FlutterFlow/flutterflow-issues#504]
Issue tracker is **ONLY** used for reporting bugs. New feature suggestions and questions should be discussed on Community or submitted through our user feedback form.

Your issue may already be reported! Please search in the [issue tracker](../) before creating one.

Please **thumbs up** this issue if you have also experienced it. You may also add more information if there is something relevant that was not mentioned. However, please refrain from comments that are not constructive, like "I have this problem too", etc.

## Expected behavior (required)

I expect the compiled project to show on the screen and be able to test

## Current behavior (required)

clicking on test mode/run mode/publish to web doesnt complete - no message appears

## To Reproduce (required)

1.load project
2.click on any of the 3 ways to test
3.
4.

### Bug Report Code (required, generate by right clicking inside Widget Tree and clicking "Get Bug Report Code")

no code

## Context (required)

unable to conmtinue

## Your environment

* Version of FlutterFlow used: 
* Platform (e.g. Web, MacOS Desktop): 
* Browser name and version: 
* Operating system and version (desktop or mobile):
latest version of flutterflow
web - microsoft windows 11
chrome

**Comment by wenkaifan0720** [comment]
Hi @masterboltdev. This could be temporary server problem. Are you still experiencing this issue? And are you experiencing it on all of your projects?

**Comment by hariprasadms** [comment]
HI @wenkaifan0720 - I can test this but I do not see any failures inconsistently. But there are build failures when local state names change which are used in actions. This issue could be one among them. 

@masterboltdev - Could you please confirm when exactly you are facing this issue.

**Comment by masterboltdev** [comment]
Originally on multiple projects.   One had built and tested clean until custom actions wouldn’t compile.  After an update where that went away, I attempted to build and it hung.   No changes made before or after to that project. Had to recompile custom actions as the builder required them to be recompiled. This project now builds. 2nd project still won’t build in test mode.  It starts with the initial building screen, then flips to the run screen with the timer counting down.  Time expires before the app is loaded. I am building a duplicate of that project - 1 small step at a time, to see when it breaks.  It broke last night so I called it a night, but it built this morning, so I’m back tracking to get it to fail again.   On the page it failed, on load, we do a backend query to load settings from a collection.  In my first project, I am doing the same, but could also access the document in an action to set defaults held in (what was) local state (now app state).    In the second project, I can’t access the document retrieved in that action and set them later in an on tap. Just checked the one that doesn’t build.  I’m trying to read the settings collection before authentication which would generate a runtime error, but I wouldn’t think would generate a build error.  Programmatically it’s correct, though logically wrong. I had moved that code in the piecemeal version so that wasn’t it.  When it broke last night, I had just added an action to copy the settings document properties into app state variables.  4 booleans and 1 reference. All 5 variables are populated.   I’ll keep on it and keep you updated on what I find.   This app has a mux integration.  It is in the one that won’t build. Not yet in the piecemeal. ThanksSent from my iPhoneOn Feb 19, 2023, at 4:12 AM, Wenkai Fan ***@***.***> wrote:﻿
Hi @masterboltdev. This could be temporary server problem. Are you still experiencing this issue? And are you experiencing it on all of your projects?
